model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -47,14 +47,20 @@ data class LicenseFinding(
      * the finding can be relied on / how confident the scanner is to be right. In most cases this is a percentage where
      * 100.0 means that the scanner is 100% confident that the finding is correct.
      */
-    val score: Float? = null
+    val score: Float? = null,
+
+    val matchedText: String? = null
 ) {
     companion object {
         val COMPARATOR = compareBy<LicenseFinding>({ it.license.toString() }, { it.location })
-            .thenByDescending { it.score }
+            .thenByDescending {
+                it.score;
+                it.matchedText
+            }
```

Comment:
This is the wrong syntax. You need to chain calls like

    .thenByDescending { it.score }.thenBy { it.matchedText }

for example. However, I don't believe that this sorting makes a lot of sense...

Reply:
Thanks for the comment, I believe I was getting some compilation issues before adding it. I will check once again if it is something we can avoid.